### PR TITLE
DED 362 Close API v3.5 and open v3.6

### PR DESCRIPTION
**Pull request by kamilatokarska**
# Deploy preview

Scroll down to the checks section for the link to the deploy preview of your branch.

# Links

Link your Jira ticket.

- [Jira]()

# Description

All the changes necessary for the switch to the new stable 3.5 version. The only thing that needs to be changed is the dates of the v3.6 in changelogs (not known for now)

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

File: src/pages/management/changelog/index.mdx [hunk 1]
```diff
@@ -23,23 +23,27 @@ The developer preview version provides a preview of the upcoming changes to the
...
 </Warning>
 
-## [v3.5] - Developer preview
+## [v3.6] - Developer preview
 
-## General
+No chnages yet.
 
-- There's a new type of request, [batch requests](/management/configuration-api/v3.5/#batch-requests). They allow for sending multiple calls of the same method in a single request.
+## [v3.5] - 2022-11-XX
```
Comment (kamilatokarska) [inline src/pages/management/changelog/index.mdx:30]: date to be confirmed

File: src/pages/messaging/agent-chat-api/changelog/index.mdx [hunk 2]
```diff
@@ -19,7 +19,11 @@ The developer preview version provides a preview of the upcoming changes to the
 
 </Warning>
 
-## [v3.5] - Developer preview
+## [v3.6] - Developer preview
+
+No changes yet.
+
+## [v3.5] - 2022-11-XX
```
Comment (kamilatokarska) [inline src/pages/messaging/agent-chat-api/changelog/index.mdx:26]: date to be confirmed

File: src/pages/messaging/customer-chat-api/changelog/index.mdx [hunk 3]
```diff
@@ -19,7 +19,11 @@ The developer preview version provides a preview of the upcoming changes to the
 
 </Warning>
 
-## [v3.5] - Developer preview
+## [v3.6] - Developer preview
+
+No changes yet.
+
+## [v3.5] - 2022-11-XX
```
Comment (kamilatokarska) [inline src/pages/messaging/customer-chat-api/changelog/index.mdx:26]: date to be confirmed

File: src/pages/data-reporting/changelog/index.mdx [hunk 4]
```diff
@@ -19,7 +19,11 @@ The developer preview version provides a preview of the upcoming changes to the
 
 </Warning>
 
-## [v3.5] - Developer preview
+## [v3.6] - Developer preview
+
+No changes yet.
+
+## [v3.5] - 2022-11-XX
```
Comment (kamilatokarska) [inline src/pages/data-reporting/changelog/index.mdx:26]:
```suggestion
## [v3.5] - 2022-11-23
```

File: src/pages/management/changelog/index.mdx (hunk 1 above)
Comment (kamilatokarska) [inline src/pages/management/changelog/index.mdx:30]:
```suggestion
## [v3.5] - 2022-11-23
```

File: src/pages/messaging/agent-chat-api/changelog/index.mdx (hunk 2 above)
Comment (kamilatokarska) [inline src/pages/messaging/agent-chat-api/changelog/index.mdx:26]:
```suggestion
## [v3.5] - 2022-11-23
```

File: src/pages/messaging/customer-chat-api/changelog/index.mdx (hunk 3 above)
Comment (kamilatokarska) [inline src/pages/messaging/customer-chat-api/changelog/index.mdx:26]:
```suggestion
## [v3.5] - 2022-11-23
```

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**